### ISSUE TITLE
how to create a right category_embeddings.npy?

### ISSUE THREAD

**Issue by yiiizuo** [Sonettoo/AeroGen#26]
hello, When I use the following encoding, the resulting npy does not match the given category_embeddings. npy

## generation embedings ##

import torch
import numpy as np
from transformers import CLIPProcessor, CLIPModel

device = "cuda" if torch.cuda.is_available() else "cpu"
model_name = "openai/clip-vit-large-patch14/"
model = CLIPModel.from_pretrained(model_name).to(device)
processor = CLIPProcessor.from_pretrained(model_name)

dota_classes = [
    "airplane",
    "airport",
    "baseballfield",
    "basketballcourt",
    "bridge",
    "chimney",
    "dam",
    "Expressway-Service-area",
    "Expressway-toll-station",
    "golffield",
    "groundtrackfield",
    "harbor",
    "overpass",
    "ship",
    "stadium",
    "storagetank",
    "tenniscourt",
    "trainstation",
    "vehicle",
    "windmill"
]

text_inputs = [f"an aerial photo of a {c}" for c in dota_classes]  # just example

inputs = processor(
    text=text_inputs, 
    return_tensors="pt", 
    padding=True, 
    truncation=True
).to(device)

with torch.no_grad():
    text_features = model.get_text_features(**inputs)

text_features_np = text_features.cpu().numpy()
print("Encoded shape:", text_features_np.shape) 

np.save("dota_clip_class_embeddings.npy", text_features_np)

## compare embedings ##

import torch
import numpy as np

raw_embeds = np.load('/mnt/zuoyi/code/remote/generation/AeroGen/datasets/category_embeddings.npy')
new_embeds = np.load('/mnt/zuoyi/code/remote/generation/AeroGen/dota_clip_class_embeddings.npy')

<img width="1828" height="734" alt="Image" src="https://github.com/user-attachments/assets/b5e3b031-f91f-4ee8-b24e-abaf42fa08c4" />

<img width="1763" height="789" alt="Image" src="https://github.com/user-attachments/assets/2de9a68a-5529-4735-8cd1-e18f5781e150" />
